java/openapi-generator-config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "apiPackage": "com.svix.generated.api",
-  "modelPackage": "com.svix.generated.model"
+  "apiPackage": "com.svix.internal.api",
+  "modelPackage": "com.svix.models"
```

Comment:
Should we have any indication that it's openapi (or generated)? and only add internal to the api

It's fine not keep as is

Reply:
I personally think its cleaner like this but i can be convinced if you think this is weird 🤷

Reply:
I don't feel strongly about it, let's keep as is